user/LICENSE

```diff
@@ -0,0 +1,191 @@
+
+                                 Apache License
```

Comment:
The file main reside only in the root dir?

Reply:
We have a license at the root, and I was considering skipping this one; wasn't sure if the NOTICE could be in the subdirectory without the license next to it, but maybe that's fine